notebooks/06-cifar10-baseline.ipynb

```diff
@@ -0,0 +1,384 @@
+{
```

Comment:
<p>lets remove the note about unresolved issue...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/pytorch-lightning/pull/4818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>